app/services/worlds_service.py

```
@@ -0,0 +1,94 @@
+        self.gw2_client = gw2_client
+
+    async def get_all_worlds(self) -> list[dict]:
+        """
+        Get all worlds from GW2 API and sync with database in the background.
+        If API fails, fallback to database.
+        """
+        try:
+            worlds_data = await self._get_currencies_from_api()
+
+            # Sync with db in background
+            asyncio.create_task(self._sync_currencies_to_db(worlds_data))
```

Ruff (0.14.1)

Store a reference to the return value of `asyncio.create_task`

(RUF006)